Slack disable updates (#152)
rapid7/insightconnect-workflows: workflows/Suspend_AD_LDAP_User_from_Slack/help.md

Sample Slack Trigger Commands:

> `@Slackbot <exampleuser>`

Reviewer: ```suggestion
`@Slackbot disable-user-ad <exampleuser>`
```

> `@Slackbot disable-user-ad <exampleuser>`

Reviewer: ```suggestion
`@Slackbot disable-user-ad <username>`
```

> `@Rapid7 InsightConnect disable-user-ad <exampleuser>`

Reviewer: ```suggestion
`@Slackbot <exampleuser>`
```

## Setup

Once the workflow has been imported, edit the workflow and setup or select your Slack connection in Slack chatbot steps.
In addition, edit the Active Directory LDAP steps with your correct Active Directory LDAP connection information. Notably, edit the search base in the `Find DN` step.

> To run the workflow, @Rapid7 InsightConnect in any channel or in a direct message

Reviewer: ```suggestion
To run the workflow, @ your Slackbot in any channel or in a direct message
```
